mappings/net/minecraft/command/argument/NbtPathArgumentType.mapping

```diff
@@ -1,12 +1,21 @@
 CLASS net/minecraft/class_2203 net/minecraft/command/argument/NbtPathArgumentType
+	FIELD field_32182 LEFT_SQUARE_BRACKET C
+	FIELD field_32183 RIGHT_SQUARE_BRACKET C
+	FIELD field_32184 LEFT_CURLY_BRACKET C
+	FIELD field_32185 RIGHT_CURLY_BRACKET C
+	FIELD field_32186 DOUBLE_QUOTE C
 	FIELD field_9898 EXAMPLES Ljava/util/Collection;
 	FIELD field_9899 NOTHING_FOUND_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
 	FIELD field_9900 INVALID_PATH_NODE_EXCEPTION Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
 	METHOD method_9352 readCompoundChildNode (Lcom/mojang/brigadier/StringReader;Ljava/lang/String;)Lnet/minecraft/class_2203$class_2210;
 		ARG 0 reader
 		ARG 1 name
+	METHOD method_9353 (Lnet/minecraft/class_2487;Lnet/minecraft/class_2520;)Z
+		ARG 1 filter
```

Comment:
This shadows the outer variable.

Reply:
Hmm. In this case I believe the outer and inner variables are the same (`computeIfAbsent` iirc?) - how do we map these?

Reply:
Wait, where is this even at? I need to check

Reply:
Oh wait, I looked at the wrong file; nevermind, I'll just un-map this for now